test/cross-browser-testing/browserstack/browserstack.karma.config.js

```diff
@@ -0,0 +1,73 @@
+const { DEBUG } = process.env;
+
+let files = [
```

Comment:
As far as I can tell, this isn't being changed, so we can make it a `const`.

```suggestion
const files = [
```